src/routes/balances/balances.controller.spec.ts

```diff
@@ -127,6 +129,57 @@ describe('Balances Controller (Unit)', () => {
       );
     });
 
+    it(`Maps native token correctly`, async () => {
+      const safeAddress = faker.finance.ethereumAddress();
+      const transactionApiBalancesResponse = [
+        balanceBuilder().with('tokenAddress', null).build(),
+      ];
+      const exchangeApiResponse = exchangeRatesBuilder()
+        .with('success', true)
+        .with('rates', { USD: 2.0 })
+        .build();
+      const chain = chainBuilder().build();
+      mockNetworkService.get.mockImplementation((url) => {
+        if (url == `https://test.safe.config/api/v1/chains/${chain.chainId}`) {
```

Comment:
I kept the url (without using faker) because I didn't want to change the overall test setup in this PR.